src/compat/posix/include/unistd.h

```diff
@@ -17,6 +17,8 @@
 #include <sys/stat.h>
 #include <sys/cdefs.h>
 
+#include <sys/uio.h>
```

Comment:
I mean, there is no this file in PR
28 апр. 2016 г. 11:02 пользователь "Denis Chusovitin" <
notifications@github.com> написал:

> In src/compat/posix/include/unistd.h
> <https://github.com/embox/embox/pull/932#discussion_r61387410>:
>
> > @@ -17,6 +17,8 @@
> >  #include <sys/stat.h>
> >  #include <sys/cdefs.h>
> >
> > +#include <sys/uio.h>
>
> This file defines iovec, so I can add delcaration of readv/writev in
> unistd.h
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/embox/embox/pull/932/files/e6f03d96cdf7e1e8d7a19585947c1a42c6b403ff#r61387410>
>